compose-ui/src/androidMain/kotlin/com/example/myapplication/root/RootPreview.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.myapplication.root
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import com.arkivanov.decompose.DefaultComponentContext
+import com.arkivanov.essenty.lifecycle.LifecycleRegistry
+import com.example.myapplication.shared.root.DefaultRootComponent
+
+/**
+ * This preview can support navigation.
+ * (Use the default Android Studio @Preview function).
+ */
+@Preview
+@Composable
+fun RootPreview() {
```

Comment:
Since this is a Kotlin Multiplatform template, having separate previews for different targets doesn't look good. There is a workaround for having the JVM (desktop) previews in `commonMain`. We can just put the following annotation in the specified package, and install Compose Multiplatform IDE Support plugin. No additional dependencies or configurations needed.

```kotlin
package androidx.compose.desktop.ui.tooling.preview

@OptIn(ExperimentalMultiplatform::class)
@OptionalExpectation
expect annotation class Preview()
```

More information here: https://x.com/arkann1985/status/1763334520546238520

Reply:
I've added the annotation and moved preview composables from jvmMain to commonMain. Meanwhile I've also removed `compose.preview` dependency and merged `@Preview`s into *Content.kt files, which makes it more concise.